### PR TITLE
ICDS Dashboard

### PR DESCRIPTION
**Pull request by lwyszomi**
@emord 
http://manage.dimagi.com/default.asp?264280
http://manage.dimagi.com/default.asp?264276
http://manage.dimagi.com/default.asp?264279

### CODE REVIEW

**Review by emord — APPROVED** [review]

**Comment by lwyszomi** [comment]
@emord build was terminated, could you kick off it?

**Comment by emord** [comment]
======================================================================
FAIL: custom.icds_reports.tests.reports.test_prevalence_of_severe:TestPrevalenceOfSevere.test_map_data
----------------------------------------------------------------------
Traceback (most recent call last):
 File "/mnt/commcare-hq-ro/custom/icds_reports/tests/reports/test_prevalence_of_severe.py", line 52, in test_map_data
 "label": "Percent of Children Underweight (0-5 years)"
AssertionError: {'rightLegend': {'info': u'Percentage of children between 0-5 years enrolled for [truncated]... != {'rightLegend': {'info': 'Percentage of children between 0-5 years enrolled for [truncated]...
Diff is 2027 characters long. Set self.maxDiff to None to see it.
----------------------------------------------------------------------

looks like a real failure

**Comment by lwyszomi** [comment]
@emord Added changes for:
https://manage.dimagi.com/default.asp?264314
https://manage.dimagi.com/default.asp?264320
https://manage.dimagi.com/default.asp?262979

File: custom/icds_reports/tests/reports/test_prevalence_of_severe.py [hunk 1]
```diff
@@ -20,42 +20,48 @@ def test_map_data(self):
...
+                        "severe": 2,
+                        "moderate": 4,
+                        "normal": 16,
+                        "total_measured": 25,
+                        "total": 497,
+                        "fillKey": "0%-5%"
                     }
                 },
-                "slug": "moderately_underweight",
-                "label": "Percent of Children Underweight (0-5 years)"
```
Comment (emord) [inline custom/icds_reports/tests/reports/test_prevalence_of_severe.py]: Why is this test changing? I don't see the complimentary change in the real code?

Comment (lwyszomi) [reply custom/icds_reports/tests/reports/test_prevalence_of_severe.py]: @emord This test used the wrong data. As you can notice in the whole test we had prevalence_of_undernutrition data, but the test is for severe.

**Comment by lwyszomi** [comment]
I added changes for: https://manage.dimagi.com/default.asp?263603

File: custom/icds_reports/tests/reports/test_prevalence_of_severe.py (hunk 1 above)
Comment (emord) [reply custom/icds_reports/tests/reports/test_prevalence_of_severe.py]: Ah. I see that now. thanks

**Review by emord — APPROVED** [review]

**Pull request merged**